openapi/SwarmDebug.yaml

```diff
@@ -553,7 +553,7 @@ paths:
       tags:
         - Chequebook
       responses:
-        "200":
+        "201":
```

Comment:
this too, @ralph-pichler

Reply:
same, returns once the tx has been sent to the backend